### ISSUE TITLE
Use the footer element

### ISSUE THREAD

**Issue by LJWatson** [w3c/Unitas#47]
The "Details" section of each page should be represented using the ```<footer>``` element. Currently it is a ```<div>``` element, which has no semantic meaning from an accessibility point of view.

When the ```<footer>``` element is used, screen readers will identify it and indicate the point where the footer content starts/ends. It also provides a hook that screen readers can use as a navigation mechanism.
